mne/annotations.py

```diff
@@ -591,12 +592,16 @@ def events_from_annotations(raw, event_id=None, regexp=None,
             else:
                 continue
         else:
-            event_id_[desc] = event_id(desc)
+            trigger = event_id(desc)
+            if trigger is not None:
+                event_id_[desc] = trigger
+            else:
+                dropped.append(desc)
 
     event_sel = [ii for ii, kk in enumerate(annotations.description)
                  if kk in event_id_]
 
-    if len(event_sel) == 0:
+    if len(event_sel) == 0 and regexp is not None:
         raise ValueError('Could not find any of the events you specified.')
```

Comment:
Does that mean there is no warning for zero events if regexp is None? If not intentional, please fix.

Reply:
I needed this otherwise it fails for an empty brainvision file that has annotations containing only